Client/Frontend/Browser/Tab.swift

```diff
@@ -348,6 +350,14 @@ class Tab: NSObject {
     func goToBackForwardListItem(_ item: WKBackForwardListItem) {
         _ = webView?.go(to: item)
     }
+    
+    func pauseAllMedia() {
+        webView?.evaluateJavaScript("pauseAll()", completionHandler: nil)
```

Comment:
Make sure `evaluateJavascript` happens on main thread

My debugger caught this

<img width="1552" alt="Zrzut ekranu 2019-04-25 o 12 09 15" src="https://user-images.githubusercontent.com/6950387/56728523-3fb0ab00-6753-11e9-8309-da891a3901ba.png">